hawc/templates/assessment-rooted.html

```diff
@@ -5,8 +5,9 @@
 {% endblock title %}
 
 {% block sidebar_outer %}
+
 <div id="sidebar-container" class="sidebar {{request.session.hideSidebar|yesno:'sidebar-collapsed, '}}"
-    data-collapsed="{{request.session.hideSidebar|yesno:"true,false"}}" data-url="{% url 'update_session' %}">
+    data-collapsed="{{request.session.hideSidebar|yesno:"true,false,empty"}}" data-url="{% url 'update_session' %}">
```

Comment:
okay I tried to do this default collapsed on mobile thing, but the yesno filter was not correctly evaluating to the None value for me (it kept defaulting to false). I did some testing using {% if request.session.hideSidebar == None %}, and that assessed the value correctly so I am stumped for why yesno isn't working